src/main/java/com/matthewglover/tictactoe/core/AlphaBeta.java

```diff
@@ -7,20 +7,21 @@
     private int alpha;
     private int beta;
 
-    public static AlphaBeta run(Game game) {
-        AlphaBeta alphaBeta = new AlphaBeta(game, 0, true, MINIMUM_ALPHA, MAXIMUM_BETA);
+
+    public static AlphaBeta run(Game game, int maxSearchDepth) {
```

Comment:
maxSearchDepth (= 6) is duplicated in at least three or four places. Can you find a better way to do this to avoid this duplication?